modules/user_accounts/php/edit_user.class.inc

```diff
@@ -780,13 +780,11 @@ class Edit_User extends \NDB_Form
             )
         );
 
-        //------------------------------------------------------------
-
         // active time windows
-
+        $dateFormat  = $config->getSetting('dateDisplayFormat');
         $dateOptions = array(
                         'language'       => 'en',
-                        'format'         => 'YMd',
+                        'format'         => $dateFormat,
```

Comment:
i believe doing this just changes what format gets appended to the date element in the LorisForm. I'm not sure if this does much in terms of changing the UI. regardless, you're making these changes for date elements (input i believe - i tracked addBasicDate function to this https://github.com/aces/Loris/blob/c75433f8dabd31b32bf5e69e659c744ec99af96e/php/libraries/LorisForm.class.inc#L270 - see addDate's doc comment) whereas this PR should change format for date displays

Reply:
i guess a good example of a date display to play with would be the dates in the data table of media, candidate_list or issue_tracker

Reply:
orrr you can maybe convert the format display on the js side using a number of functions in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date